app/src/main/cpp/skyline/vfs/partition_filesystem.cpp

```diff
@@ -19,12 +19,12 @@ namespace skyline::vfs {
         size_t stringTableOffset = sizeof(FsHeader) + (header.numFiles * entrySize);
         fileDataOffset = stringTableOffset + header.stringTableSize;
 
-        std::vector<char> stringTable(header.stringTableSize);
+        std::vector<char> stringTable(header.stringTableSize + 1);
         backing->Read(stringTable.data(), stringTableOffset, header.stringTableSize);
```

Comment:
You should explicitly do `stringTable[header.stringTableSize] = 0;`. `std::vector` doesn't 0-initialize by default, nor do we want to do that here as most of it is overwritten.

Reply:
`stringTable[header.stringTableSize] = 0;` doesn't work. It throws an error `Variable-sized object may not be initialized`. Should I just memset the vector with `0` or is there a better alternative?